packages/radio-group/src/Radio/Radio.tsx

```diff
@@ -55,6 +56,7 @@ function Radio({
   checked,
   size = Size.Default,
   description,
+  bold,
```

Comment:
We should add the `bold` prop to the single `Radio` story

Reply:
Added